firefox-ios/Client/Frontend/TrackingProtection/BlockedTrackersHelpers/BlockedTrackersModel.swift

```diff
@@ -0,0 +1,68 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+struct BlockedTrackersTableModel {
+    let topLevelDomain: String
+    let title: String
+    let URL: String
+    let contentBlockerStats: TPPageStats?
+    let connectionSecure: Bool
+
+    func getItems() -> [BlockedTrackerItem] {
+        let crossSiteCount = String(contentBlockerStats?.getTrackersBlockedForCategory(.advertising) ?? 0)
+        let fingerprintersCount = String(contentBlockerStats?.getTrackersBlockedForCategory(.fingerprinting) ?? 0)
+        let socialMediaCount = String(contentBlockerStats?.getTrackersBlockedForCategory(.social) ?? 0)
+        let trackingContentCount = String(contentBlockerStats?.getTrackersBlockedForCategory(.analytics) ?? 0)
```

Comment:
wasn't planning on putting it in the middleware because those are just static numbers that would be available in the model, it's the equivalent of having a label init with a predetermined value.

Reply:
Oh are we not having the model instantiate in the middleware? Curious where your thoughts were with that, not part of this PR though.